serde/core.py

```diff
@@ -413,7 +413,7 @@ def is_list_instance(obj: Any, typ: Type[Any]) -> bool:
 
 
 def is_set_instance(obj: Any, typ: Type[Any]) -> bool:
-    if not isinstance(obj, set):
+    if not isinstance(obj, (set, frozenset)):
```

Comment:
ah, fronzenset is not derived from set. Interesting

Reply:
Yeah I wasn't aware of this myself.